ui/src/stylesheets/admonitionblock.scss

```
@@ -8,31 +8,26 @@
 
 .admonitionblock,
 .alert {
-  border: 1px solid rgb(196, 196, 196);
-  border-radius: var(--border-radius);
-  font-family: var(--heading);
-  color: rgb(36, 36, 36);
-  line-height: 100%;
   max-width: 90vw;
+  background-color: white;
+  border: solid #ddd;
+  border-radius: var(--border-radius);
+  border-width: 1px 1px 1px 6px;
+  font-family: var(--sans-serif);
+  font-weight: var(--weight-normal);
+  color: var(--color-text-light);
   font-size: 0.9rem;
   text-align: center;
-  line-height: 1.4;
-  font-weight: var(--weight-medium);
-  padding: 1rem 0.9rem;
-  display: block;
-  background-color: white;
+  line-height: 1.5;
+  padding: 1rem .9rem;
   margin: 1.5rem 0;
-  background-repeat: no-repeat;
-  background-size: 32px;
-  background-origin: border-box;
-  background-position: 1.4rem center;
-  
+  word-break: break-word;
```

[According to MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#Values), `word-break: break-word` is deprecated, and it is equivalent to `word-break: normal` (the default) combined with `overflow-wrap: break-word`.

```suggestion
  overflow-wrap: break-word;
```

---

Sure, thanks Fran